### ISSUE TITLE
False positive (probably rounding on reflect)

### ISSUE THREAD

**Issue by Simoris** [Alya-N/C-Hero-Calc#43]
```
Solution for [Followers:       0 | ahirate:1000 ashygu:1000 aauri:1000 aneptunius:1000 cupid:1000 <==]:
  [Followers:       0 | masterlee:99.6 hidoka:99.6 leprechaun:99.5 guy:99.5 raiderrose:99.5 <==]

  1426748 Fights simulated.
  Total Calculation Time: 0
  Calc Version: 4.7.0.2test_broken_res_3

Battle Replay (Use on Ingame Tournament Page):
470tAAs2PWnHrBjuCx3CY6AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAL5oF20CiaBedAymgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==
```

Only draws instead of winning. The enemy aHirate survives Lee's first reflect with 1 hp, so the (revived) Lee has to take a second turn and dies, so the battle only yields a draw.

**Comment by Alya-N** [comment]
Ah, yeah, I actually fixed it in the version I'll release after revive is fixed. I don't think I'll make a new calc version just for a rounding fix, but there are some more issues I can fix and put in the next version if revive isn't fixed sooner.

**Comment by Simoris** [comment]
Yeah, that's probably fine (as long as Gaia manages to get his fix done soonish). Thanks for keeping this up.

**Comment by Alya-N** [comment]
Okay, it should now be fixed in version **4.7.0.2test_broken_res_4** which shows a new solution with those heroes:
```
Solution for [Followers:       0 | ahirate:1000 ashygu:1000 aauri:1000 aneptunius:1000 cupid:1000 <==]:
  [Followers:       0 | hidoka:99.6 masterlee:99.6 guy:99.5 leprechaun:99.5 raiderrose:99.5 <==]

  296 Fights simulated.
  Total Calculation Time: 0
  Calc Version: 4.7.0.2test_broken_res_4

Battle Replay (Use on Ingame Tournament Page):
470fAAtOPWbHsFjtgx3CY6AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAL5oF20CiaBedAymgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==
```
